e2e_test/slow_tests/ddl/creation_check.slt

```diff
@@ -0,0 +1,22 @@
+statement ok
+create table t(v1 int);
+
+statement ok
+insert into t select 1 from generate_series(1, 10000);
+
+# trigger high barrier latency
+system ok
```

Comment:
Instead of using system commands, can we adopt `connection <tag>` to achieve this?

Reply:
Unfortunately no, in scenarios with high barrier latency, the initial creation of table `t2` will be blocked. We have to spawn and make it running in background.